win32ss/pch.h

```diff
@@ -103,8 +104,8 @@ extern "C" {
 #include <ft2build.h>
 #include FT_FREETYPE_H
 
-#define InterlockedIncrementUL(Value) InterlockedIncrement((PLONG)Value)
-#define InterlockedDecrementUL(Value) InterlockedDecrement((PLONG)Value)
+#define InterlockedIncrementUL(Value) (ULONG)InterlockedIncrement((PLONG)(Value))
+#define InterlockedDecrementUL(Value) (ULONG)InterlockedDecrement((PLONG)(Value))
```

Comment:
The (ULONG) changes might make sense, but the added braces around `Value` do add no value.